### PR TITLE
docs: 1.0.3 changelog

### PR DESCRIPTION
**Pull request by ryane**
[skip ci]

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+
+* All OS packages installed from central repositories #1193, #1204
+* Elasticsearch and Kibana UIs are now available in Mantl UI #1175
+* Default Kibana dashboard imported by default #1139
+* ELK improvements #1191
+* Addon: iptables role to restrict network traffic within the cluster #593
+* Improved documentation #1106, #1153, #1160, #1161, #1163, #1164, #1169, #1181, #1222, #1227
+* Update terraform.py #1105
+* Mantl rename: references to microservices-insfrastructure replaced with Mantl #1134
+* Pull request and issue templates #1137
```
Comment (BrianHicks) [inline CHANGELOG.rst]: This seems super internal and super minor. Should it really go in a changelog?

**Comment by BrianHicks** [comment]
Could we maybe reword the updates to be more descriptive? Something like the form "Verbed noun, affects you because x. #1234"

I recognize we haven't been doing this so take this advice with a grain of salt if this is super time-sensitive.

**Comment by ryane** [comment]
What we've been doing is using the changelog as just a simple list of prs that are included in the release. The (forthcoming) release notes will include higher-level, more user-friendly descriptions of what's in the release.

**Comment by BrianHicks** [comment]
:shipit:
